docs/source/markdown/podman-run.1.md

```diff
@@ -992,8 +992,8 @@ Note: Labeling can be disabled for all containers by setting label=false in the
 
 - **no-new-privileges**: Disable container processes from gaining additional privileges
 
-- **seccomp=unconfined**: Turn off seccomp confinement for the container
-- **seccomp**=_profile.json_: Allowed syscall list seccomp JSON file to be used as a seccomp filter
+- **seccomp=unconfined**: Turn off seccomp confinement for the container.
+- **seccomp=profile.json**:  White listed syscalls seccomp Json file to be used as a seccomp filter. Note that the `io.podman.annotations.seccomp` annotation is set with the specified value as shown in `podman inspect`.
```

Comment:
```suggestion
- **seccomp=profile.json**:  Whitelisted syscalls seccomp JSON file to be used as a seccomp filter. Note that the `io.podman.annotations.seccomp` annotation is set with the specified value as shown in `podman inspect`.
```

Reply:
since we are at it, should we `s|white|allow|`?

Reply:
I think it should be `JSON file to be used as a seccomp filter` since it could be any kind of filter.